_includes/footer.html

```diff
@@ -0,0 +1,55 @@
+<footer class="site-footer">
+
+  <div class="wrapper">
+
+    <h2 class="footer-heading">{{ site.title }}</h2>
+
+    <div class="footer-col-wrapper">
+      <div class="footer-col  footer-col-1">
+        <ul class="contact-list">
+          <li>{{ site.title }}</li>
+          <li><a href="mailto:{{ site.email }}">{{ site.email }}</a></li>
```

Comment:
This is going to be broken right now if you are actually including it. If you aren't including it, that's fine but you should probably cut out the stuff you aren't using.